display/display.go

```diff
@@ -28,6 +28,12 @@ var termH int
 // The user input and URL display bar at the bottom
 var bottomBar = cview.NewInputField()
 
+var originalText []byte
+var searchBar = cview.NewInputField()
```

Comment:
Why a separate bar instead of using the existing one? Also this primitive would need to be themed like the bottom bar is. But I think just keeping one bar is better.